File: linmour-order-module/linmour-order-api/src/main/java/com/linmour/order/pojo/Dto/ShopListDto.java
package com.linmour.order.pojo.Dto;

import lombok.Data;

import java.io.Serializable;
import java.math.BigDecimal;

@Data
public class ShopListDto implements Serializable {
    private Long id;

    /**
     * 商品名字
     */

    private String name;
    /**
     * 店铺id
     */

    private Long shopId;

    /**
     * 商品简介
     */

    private String intro;

    private BigDecimal price;

    /**
     *
     */

    private Long specId;

    /**
     * 商品状态 0下架 1上架
     */

    private Boolean status;

    /**
     * 商品图
     */

    private String picture;

    private Long sortId;


    private Integer valueSpec;
    private Integer nonValueSpec;

    private Integer number;
}

File: linmour-order-module-go/server/models/ShopListDto.go
package models

type ShopListDto struct {
	ID      int64   `json:"id"`
	Name    string  `json:"name"`
	ShopId  int64   `json:"shopId"`
	Intro   string  `json:"intro"`
	Price   float64 `json:"price"`
	SpecId  int64   `json:"specId"`
	Status  bool    `json:"status"`
	Picture string  `json:"picture"`
	SortId  int64   `json:"sortId"`
	Number  int     `json:"number"`
}
